lib/kafka/schema_registry/service.rb

```diff
@@ -58,38 +62,27 @@ def compatible?(subject, schema, version = 'latest')
       # - nil if the subject or version does not exist
       # - a list of compatibility issues
       # https://docs.confluent.io/platform/current/schema-registry/develop/api.html#sr-api-compatibility
-      def compatibility_issues(subject, schema, version = 'latest')
-        data = post("/compatibility/subjects/#{subject}/versions/#{version}",
-                    expects: [200, 404],
-                    body: { schema: schema.to_s }.to_json, query: { verbose: true }, idempotent: true)
+      def compatibility_issues(topic, schema, version = 'latest')
+        data = request("/compatibility/subjects/#{topic}-value/versions/#{version}",
+                       method: :post,
+                       expects: [200, 404],
+                       body: { schema: schema.to_s }.to_json, query: { verbose: true }, idempotent: true)
 
         data.fetch('messages', []) unless data.key?('error_code')
       end
 
       # Get global config
       def global_config
-        get('/config', idempotent: true)
+        request('/config', idempotent: true)
       end
 
       # Get config for subject
-      def subject_config(subject)
-        get("/config/#{subject}", idempotent: true)
+      def subject_config(topic)
+        request("/config/#{topic}-value", idempotent: true)
       end
 
       private
 
-      def get(path, **)
```

Comment:
A recommendation by Steven Cummings to remove these as they don't offer much

Reply:
That's a great change!